template/webpack.config.dev.js

```diff
@@ -44,19 +41,19 @@ module.exports = {
   ],
 
   module: {
-    loaders: [
+    rules: [
       {
         test: /\.scss$/, // sass files
-        loader: 'style!css!autoprefixer?browsers=last 2 version!sass?outputStyle=expanded',
+        loaders: ['style-loader', 'css-loader?sourceMap', 'sass-loader?sourceMap'],
       },
       {
         test: /\.(ttf|eot|svg|woff)(\?[a-z0-9]+)?$/, // fonts files
-        loader: 'file-loader?name=[path][name].[ext]',
+        loaders: ['file-loader?name=[path][name].[ext]'],
```

Comment:
`loaders` will need to be changed to `use` where specified everywhere in this file to comply with `webpack` 4 AFAIK